listenbrainz/model/user.py

```diff
@@ -47,3 +52,11 @@ class UserAdminView(AdminModelView):
         'musicbrainz_id',
         'musicbrainz_row_id',
     ]
+
+    def delete_model(self, model):
+        try:
+            delete_user(model.musicbrainz_id)
```

Comment:
I'm a bit surprised that all of these methods use a username instead of a lb row id or musicbrainz row id... might be worth adding a ticket to clean this up. I guess it also relies on LB-866

Reply:
user_name are in use *everywhere*. :(

Reply:
Yeah, probably makes sense to use an id. Will open a ticket.